src/crashlog.cpp

```diff
@@ -267,9 +267,15 @@ char *CrashLog::LogLibraries(char *buffer, const char *last) const
 #endif /* WITH_PNG */
 
 #ifdef WITH_SDL
-	const SDL_version *v = SDL_Linked_Version();
-	buffer += seprintf(buffer, last, " SDL:        %d.%d.%d\n", v->major, v->minor, v->patch);
-#endif /* WITH_SDL */
+	const SDL_version *sdl_v = SDL_Linked_Version();
+	buffer += seprintf(buffer, last, " SDL:        %d.%d.%d\n", sdl_v->major, sdl_v->minor, sdl_v->patch);
+#else
+#ifdef WITH_SDL2
```

Comment:
#elif ? Otherwise this becomes a bit of a weird statement :D (and the comment on the `#endif` is in the wrong place for sure)